### PR TITLE
Move fabric module from client_sdk to sdk

### PR DESCRIPTION
**Pull request by Ram-srini**
1. avalon_sdk is single sdk for all clients and moved fabric to sdk.
2. Fix the lint errors in fabric modules.

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: sdk/avalon_sdk/fabric/eventlistener.py [hunk 1]
```diff
@@ -84,16 +87,18 @@ def saveConfig(self):
 
     async def startEventHandling(self):
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/eventlistener.py]: Please use PEP8 for function names. startEventHandling -> start_event_handling

File: sdk/avalon_sdk/fabric/eventlistener.py [hunk 2]
```diff
@@ -84,16 +87,18 @@ def saveConfig(self):
...
         def eventHandler(event, block_num, txnid, status):
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/eventlistener.py]: eventHandler -> event_handler

File: sdk/avalon_sdk/fabric/txcommiter.py [hunk 3]
```diff
@@ -27,6 +28,7 @@
 
 logger = logging.getLogger(__name__)
 
+
 class TxCommitter(base.ClientBase):
 
     def getEndorsers(self, queryonly=False):
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/txcommiter.py]: getEndorsers -> get_endorsers

File: sdk/avalon_sdk/fabric/txcommiter.py [hunk 4]
```diff
@@ -42,27 +44,30 @@ def getEndorsers(self, queryonly=False):
...
     # fcn - chaincode function name to be invoked
     # cc_version - chaincode version to be used
-    # queryonly - if the invocation does not result in ledger change, queryonly should be
-    #             set to True, if the invocation does result in ledger change, it should
-    #             be set to False.
+    # queryonly - if the invocation does not result in ledger change,
+    # queryonly should be set to True,
+    # if the invocation does result in ledger change, it should
+    # be set to False.
     def ccInvoke(self, args, cc_name, fcn, cc_version, queryonly=False):
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/txcommiter.py]: ccInvoke -> cc_invoke

File: sdk/avalon_sdk/fabric/txcommiter.py [hunk 5]
```diff
@@ -42,27 +44,30 @@ def getEndorsers(self, queryonly=False):
         return endorsers
 
     # Invoke a chaincode method
-    # txData - the json serialized data used as the sole parameter to invoke the chaincode
+    # txData - the json serialized data used as the
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/txcommiter.py]: txData is not present in argument list

File: sdk/avalon_sdk/fabric/eventlistener.py [hunk 6]
```diff
@@ -22,15 +22,18 @@
 
 logger = logging.getLogger(__name__)
 
+
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/eventlistener.py]: Please use PEP8 naming convention. rename this file eventlistener.py -> event_listener.py

File: sdk/avalon_sdk/fabric/txcommiter.py [hunk 7]
```diff
@@ -18,7 +18,8 @@
 import json
```
Comment (manojgop) [inline sdk/avalon_sdk/fabric/txcommiter.py]: rename thie file txcommiter.py -> tx_commiter.py

File: sdk/avalon_sdk/fabric/eventlistener.py (hunk 1 above)
Comment (Ram-srini) [reply sdk/avalon_sdk/fabric/eventlistener.py]: Why can't we enable pep8 standard in lint to catch these errors?

Comment (manojgop) [reply sdk/avalon_sdk/fabric/eventlistener.py]: We can by following http://pycodestyle.pycqa.org/en/latest/intro.html. But we need to ensure that existing files are following PEP8 top avoid breaking the build due to lint issues.  So first we need to raise a separate PR fixing PEP8 issues in existing files.

Comment (danintel) [reply sdk/avalon_sdk/fabric/eventlistener.py]: @Ram-srini: we can enable PEP8 in `run_lint`. The problem is the current code is not PEP8 compliant and requires fixing.

Comment (Ram-srini) [reply sdk/avalon_sdk/fabric/eventlistener.py]: Done.

**Comment by Ram-srini** [comment]
@manojgop Fixed pep8 name convention errors.

**Pull request merged**